moped-database/hasura-cluster

```diff
@@ -27,12 +27,7 @@ function __get_platform() {
 
 function __set_postgresql_image() {
```

Comment:
Really glad to see the multiarch image introduced here now!! Out of curiosity and definitely for another day, but do you think we need to keep around any of these helpers to get the platform and set the image in the long term? I can make a tech debt issue to do this later.

Reply:
Good call Mike - I thought about that, and waffled back and forth...

I think it's right to yoink them out of there and clean that up, but I had that urge to keep them "in case we needed them" -- but that is a silly instinct in the land of `git`. I appreciate you making that issue - it'll be a quick one.

Reply:
cool - that makes sense to me too. i also had that thought about keeping it just in case!